Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -4290,7 +4290,7 @@ MediaSessionPlaylistEnabled:
 
 MediaSourceEnabled:
   type: bool
-  status: stable
+  status: embedder
```

Comment:
How about internal?
That would allow easier switch off/on of this feature.

Reply:
There is code already in place to handle the MSE flag so that it shows in the internal flag list with the other media flags